docs/api-reference/layers/text-layer.md

```diff
@@ -173,7 +173,7 @@ Available options are `break-all` and `break-word`. A valid `maxWidth` has to be
 
 * Default: `-1`
 
-`maxWidth` is used together with `break-word` for wrapping text. The value of `maxWidth` specifies the width limit to break the text into multiple lines.
+`maxWidth` is used together with `break-word` for wrapping text. The value of `maxWidth` is multiplied with the text size to specify the width limit to break the text into multiple lines. For example, `maxWidth: 10.0` used with `getSize: 12` is roughly the equivalent of `max-width: 120px` in CSS.
```

Comment:
Perhaps one option could be to add a new prop `relativeMaxWidth` to avoid breaking the old one, and instead deprecating that one. Also the proposed name signals the "subtleness" of the prop a bit better?

Reply:
I am not a fan of the word `relative`. Another example is the prop `lineHeight`, which is also a multiplier of text size.

Essentially all layout props of this layer must be relative, i.e. scale with the text size. Because text size can be zoom-dependent, reflowing characters at zoom change is not an option.

Reply:
Hmm, yes, if it works the same way for all props then we should not make any special name here. Perhaps a section about sizing at the top of the doc could make sense, rather than having to piece it together from the prop docs.